src/views/Company.vue

```diff
@@ -1,20 +1,313 @@
 <template>
-  <div class="company-page" :style="{marginTop : `-${navbarHeight}px`}">
-    <h1>Company Page</h1>
+  <div
+    class="company-page pt-100"
+    style="background-color: #eaebf9"
+    :style="{ marginTop: `-${navbarHeight}px` }"
+  >
+    <v-container>
+      <v-row>
+        <v-col cols="12">
+          <h1 class="text-h2 font-weight-600">
+            Disrupting the Banking Sector with Competitive Services
+          </h1>
+          <section class="mt-16">
+            <slick ref="slick" :options="slickOptions">
```

Comment:
Again, we will not be using the vue-slick package for this carousel. Use [Slider Groups](https://vuetifyjs.com/en/components/slide-groups/) instead.